htdocs/ecm/class/ecmfiles.class.php

```diff
@@ -165,10 +166,10 @@ public function create(User $user, $notrigger = false)
 			 $this->extraparams = trim($this->extraparams);
 		}
 		if (isset($this->fk_user_c)) {
-			 $this->fk_user_c = trim($this->fk_user_c);
+			 $this->fk_user_c = $this->fk_user_c;
 		}
 		if (isset($this->fk_user_m)) {
-			 $this->fk_user_m = trim($this->fk_user_m);
+			 $this->fk_user_m = $this->fk_user_m;
```

Comment:
New code seems useless.
May be you want to do 
$this->fk_user_m = (int) $this->fk_user_m
?